marketplacetests/payment/app.py

```diff
@@ -39,6 +42,14 @@ def app_name(self):
         return self.marionette.find_element(*self._app_name_locator).text
 
     @property
+    def in_app_product_name(self):
+        return self.marionette.find_element(*self._in_app_product_name_locator).text
+
+    @property
+    def confirm_payment_panel_text(self):
```

Comment:
I'm not sure why the property is called `confirm_payment_panel_text` but the locator is called `_confirm_payment_header_locator`. This is confusing. I suggest being consistent with the names, whichever one is more appropriate.